svc-bip-api/src/main/java/gov/va/vro/bip/service/IBipApiService.java

```diff
@@ -1,65 +1,28 @@
 package gov.va.vro.bip.service;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
 import gov.va.vro.bip.model.BipClaim;
 import gov.va.vro.bip.model.BipUpdateClaimResp;
 import gov.va.vro.bip.model.ClaimContention;
 import gov.va.vro.bip.model.ClaimStatus;
 import gov.va.vro.bip.model.UpdateContentionReq;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
 
 import java.util.List;
 
-/**
- * BIP Claims API service.
- *
- * @author warren @Date 11/8/22
- */
 public interface IBipApiService {
+  @RabbitListener(queues = "getClaimDetailsQueue", errorHandler = "errorHandlerForGetClaimDetails")
```

Comment:
These @RabbitListener annotations really should not be in this interface. This would mean this interface is directly tied to implementations reliant on rabbitMQ. Just move them to the implementation's methods.

Reply:
Good point.  I agree. So in the next iteration, there will be a Controller layer decoupling the service from all the RMQ transport stuff.